lib/dnsimple/client/templates_records.rb

```diff
@@ -0,0 +1,123 @@
+module Dnsimple
+  class Client
+    module TemplatesRecords
+
+      # Lists the records in the template.
+      #
+      # @see https://developer.dnsimple.com/v2/templates/records/#list
+      # @see #all_records
+      #
+      # @example List the first page of records for the template "alpha"
+      #   client.templates.records(1010, "alpha")
+      #
+      # @example List records for the template "alpha", providing a specific page
+      #   client.templates.records(1010, "alpha", query: { page: 2 })
+      #
+      # @param  [Fixnum, Dnsimple::Client::WILDCARD_ACCOUNT] account_id the account ID or wildcard
+      # @param  [String] template_id the template name
+      # @param  [Hash] options
+      # @return [Dnsimple::PaginatedResponse<Dnsimple::Struct::TemplateRecord>]
+      #
+      # @raise  [Dnsimple::RequestError]
```

Comment:
All these methods also raise `@raise  [Dnsimple::NotFoundError]` when the template is not found.